code/html/index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width , initial-scale=1">
+
+<head>
+  <script src="D:/calculator/code/js/script-math.js"></script>
```

Comment:
Don't remember that you push this change into production and whole application will fail. Therefore we have code review.

Reply:
relative addressing used, due to creation of folder structure, new file called index.html added as introductory tool